spring-integration-file/src/main/java/org/springframework/integration/file/tail/OSDelegatingFileTailingMessageProducer.java

```diff
@@ -35,7 +36,7 @@
  *
  */
 public class OSDelegatingFileTailingMessageProducer extends FileTailingMessageProducerSupport
-		implements Runnable {
+		implements SchedulingAwareRunnable {
```

Comment:
The concern that any `Runnable` can be run externally form user-specific `Executor`.
The main concern do not provoke users for unexpected results :smile: